StreamChatClient_v3/Database/DatabaseSession.swift

```diff
@@ -0,0 +1,46 @@
+//
+// DatabaseSession.swift
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+
+extension NSManagedObjectContext: DatabaseSession {}
+
+protocol DatabaseSession {
```

Comment:
Would be nice to have a rule to add comments in the code for protocols.

Reply:
Definitely, this is still a WIP though.